src/core/Config.cpp

```diff
@@ -127,7 +127,7 @@ static const QHash<Config::ConfigKey, ConfigDirective> configStrings = {
     // Security
     {Config::Security_ClearClipboard, {QS("Security/ClearClipboard"), Roaming, true}},
     {Config::Security_ClearClipboardTimeout, {QS("Security/ClearClipboardTimeout"), Roaming, 10}},
-    {Config::Security_ClearSearch, {QS("Security/ClearSearch"), Roaming, true}},
+    {Config::Security_ClearSearch, {QS("Security/ClearSearch"), Roaming, false}},
```

Comment:
Wondering, why no failed GUI tests where this option is selected by default?
Probably we don't have a test with default values and test configs are predefined.
![SelectedClearSearchOption](https://user-images.githubusercontent.com/121412908/220829868-dd9b51f1-0ef8-4188-97cc-36e2f8ab9dfa.png)

Reply:
The gui tests are brittle enough. I don't care to test  things like that, especially non-critical features.